### ISSUE TITLE
allow custom indent levels for chained expression

### ISSUE THREAD

**Issue by millermedeiros** [millermedeiros/esformatter#188]
```js
this.headers
  .attr({
    "aria-selected": "false",
    "aria-expanded": "false",
    tabIndex: -1
  })
  // notice that "next()" creates a new indent level
  .next()
    .attr({
      "aria-hidden": "true"
    })
    .hide();
```

test case borrowed from https://github.com/jzaefferer/esformatter/commit/127d02fc37473dbca0d98126f9c638639c9b74c2

see #146

**Comment by millermedeiros** [comment]
reopening since we decided to not close `plugins-wishlist` bugs until they are implemented

**Comment by jzaefferer** [comment]
I wrote a plugin for this. Need to polish and publish it.

**Comment by jzaefferer** [comment]
Published 1.0.0 of my plugin to npm: https://www.npmjs.com/package/esformatter-jquery-chain

Also added the plugin the [wiki](https://github.com/millermedeiros/esformatter/wiki/Plugins).

**Comment by jzaefferer** [comment]
Btw. the plugin, as the title implies, is specific for the jQuery style guide. But it should be a good reference in case anyone else needs something like it, and it should also be pretty reasonable to add options for the methods that in- or decrease indent.
